Documentation/getting-started.md

```diff
@@ -0,0 +1,164 @@
+  resources:
+  - ingresses
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+```
+
+Now, create a ClusterRoleBinding to bind the ClusterRole to the Prometheus ServiceAccount.
+
+```yaml mdox-exec="cat example/rbac/prometheus/prometheus-cluster-role-binding.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
+- kind: ServiceAccount
+  name: prometheus
+  namespace: default
+```
+
+Apply all these manifests to create the necessary RBAC resources. Now you are all set to deploy a Prometheus instance. Here is an example of a basic Prometheus instance manifest.
+
+```yaml mdox-exec="cat example/user-guides/getting-started/prometheus.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prometheus
+spec:
+  serviceAccountName: prometheus
+  resources:
+    requests:
+      memory: 400Mi
+```
+
+To verify that the instance is up and running, run:
+
+```bash
+kubectl get -n default prometheus prometheus -w
+```
+
+For more information, see the [Prometheus Operator RBAC guide]({{< ref "rbac" >}}).
+
+## Deploying Alertmanager
+
+Let us take a simple example that creates 3 replicas of Alertmanager.
+
+```yaml mdox-exec="cat example/user-guides/alerting/alertmanager-example.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: example
+spec:
+  replicas: 3
+```
+
+Wait for all Alertmanager pods to be ready:
+
+```bash
+kubectl get pods -l alertmanager=example -w
+```
+
+However, Alertmanager as it is now is of no use to us. To properly use Alertmanager, it is important to understand the relationship between Prometheus and Alertmanager. Alertmanager is used to:
+
+* Deduplicate alerts received from Prometheus.
+* Silence alerts.
+* Route and send grouped notifications to various integrations (PagerDuty, OpsGenie, mail, chat, …).
+
+So, to put Alertmanager instances to use, you would need to integrate it with Prometheus.
+
+## Integrating Alertmanager With Prometheus
+
+This Alertmanager cluster is now fully functional and highly available, but no
+alerts are fired against it.
+
+First, create a Prometheus instance that will send alerts to the Alertmanger cluster:
+
+```yaml mdox-exec="cat example/user-guides/alerting/prometheus-example.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: example
+spec:
+  serviceAccountName: prometheus
+  replicas: 2
+  alerting:
+    alertmanagers:
+    - namespace: default
+      name: alertmanager-example
+      port: web
+  serviceMonitorSelector:
+    matchLabels:
+      team: frontend
+  ruleSelector:
+    matchLabels:
+      role: alert-rules
+      prometheus: example
+```
+
+The `Prometheus` resource discovers all of the Alertmanager instances behind
+the `Service` created before (pay attention to `name`, `namespace` and `port`
```

Comment:
In this example, they are using a Service created from [this](https://prometheus-operator.dev/docs/developer/alerting/#exposing-the-alertmanager-service) section. Should I add that section before this part?

Reply:
As we discussed we will add a new page to the web site based on this document: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/exposing-prometheus-and-alertmanager.md

So we can reference it on the required places like in this section.

Reply:
Will do it in the next PR.